week-1/I-floats/exercise.js

```diff
@@ -1,2 +1,9 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var totalOfPeople = numberOfStudents + numberOfMentors;
+var perOfStudents = (numberOfStudents / totalOfPeople) * 100;
+var perOfMentors = (numberOfMentors / totalOfPeople) * 100;
+var perRooughMentors = Math.round(perOfMentors);
+var perRoughStudents = Math.round(perOfStudents);
+console.log("Persentage students: " + perRoughStudents + "%");
+console.log("Persentage mentors: " + perRooughMentors + "%");
```

Comment:
very good again. I would only add be careful to tidy typos up (misspelled variable names) e.g. RooughMentors so that code is consistent and communicates clearly.

Reply:
you might be interested to read up on similar Math functions to round() . There is ceil() and floor(). https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/ceil These are useful to know of, depending on what we want. ("Rounding up or down?")